ts/features/bonus/cgn/components/detail/CgnStatusDetail.tsx

```diff
@@ -87,16 +85,35 @@ const elementsFromStatus = ({ cgnDetail }: Props): StatusElements => {
                 )}
               </H5>
             )}
+            {renderRowBlock(
+              <H5
+                weight={"Regular"}
+                color={"bluegrey"}
+                testID={"expiration-date-label"}
+              >
+                {I18n.t("bonus.cgn.detail.status.expiration.cgn")}
+              </H5>,
+              <H5
+                weight={"SemiBold"}
+                color={"bluegrey"}
+                testID={"expiration-date-value"}
+              >
+                {localeDateFormat(
+                  cgnDetail.expiration_date,
+                  TypedI18n.t("global.dateFormats.shortFormat")
```

Comment:
```suggestion
                  I18n.t("global.dateFormats.shortFormat")
```
there are other of `TypedI18n` usages inside this file